_data/changelogs/docs-settings.yml

```diff
@@ -2,6 +2,12 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```

Comment:
## Note

Will need to update before merge.

Reply:
Added suggestion